pkg/upgradeservice/preflight/preflight.go

```diff
@@ -195,3 +230,10 @@ func setPreflightData(preflightData *PreflightData) error {
 	}
 	return nil
 }
+
+func ResetPreflightData() error {
+	if err := os.RemoveAll(PreflightDataFilepath); err != nil {
+		return errors.Wrap(err, "failed to remove preflight data")
```

Comment:
maybe don't fail if it doesn't exist?

Reply:
I believe [`os.RemoveAll`](https://pkg.go.dev/os#RemoveAll) will do this. Just switched it in https://github.com/replicatedhq/kots/pull/4693/commits/dbd8cd02a9acf4a9e7dd2828d9dd38030895647a for this same reason.